scaletest/templates/scaletest-runner/scripts/lib.sh

```diff
@@ -271,18 +270,18 @@ fetch_coder_full() {
 	ns=$(namespace)
 	if [[ -z "${ns}" ]]; then
 		log "Could not determine namespace!"
-		exit 1
+		return
```

Comment:
We can still indicate error here by returning a non-zero value, otherwise the return code of `log` will be implicitly used (which is `0`).

```suggestion
		return 1
```